bin/steps/python

```diff
@@ -97,6 +97,15 @@ if [ ! "$SKIP_INSTALL" ]; then
   hash -r
 fi
 
+# Heroku uses the get-pip utility maintained by the Python community to vendor Pip.
+# https://github.com/pypa/get-pip
+GETPIP="https://lang-python.s3.amazonaws.com/etc/get-pip.py"
```

Comment:
Could you add the SHA to the filename here? and check the SHA after downloading?

```
GETPIP="https://lang-python.s3.amazonaws.com/etc/get-pip-${SHA}.py"
```

This ensures that the bytes you download are the same ones you are expecting, and that when you need to update it, that is a distinct PR that goes through code review before getting released.